gateway/main/urls.py

```diff
@@ -22,6 +22,7 @@
 from drf_yasg.views import get_schema_view
 from drf_yasg import openapi
 import probes.views
+import version.views as version
```

Comment:
Maybe just:

```python
import version.views
```

So we can use later:

```python
path("version/", version.views.version, name="version"),
```

And maintain the same format than `probes`.

Reply:
Thanks!  I update.